config/routes.rb

```diff
@@ -9,7 +9,9 @@
 
```

Comment:
![image](https://github.com/devalibello/blog-app/assets/93039359/a57630a5-ede4-4841-b309-05a1b8d1a107)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you made the image squared because right now it looks more lengthy and less in width 😄 Kindly make it in equal width and length to make your view look professional 👍